holy-grail-backend/backend/app/tasks/ad_analytics_loop.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+import requests
+
+from app.utils.worker import celery_app
+
+
+@celery_app.task(name="ad_analytics_loop")
+def ad_analytics_loop() -> None:
+    resp = requests.post(
+        f"http://{os.environ['BACKEND_CONTAINER_URL']}/ad_analytics/new_day"
```

Comment:
idk if my config is wrong but the ``os.environ['BACKEND_CONTAINER_URL']`` should be ``holy-grail-backend:8005`` instead of ``backend:8005``

Reply:
this also impacts the health check task